osquery/remote/enroll/enroll.cpp

```diff
@@ -56,9 +62,22 @@ std::string getNodeKey(const std::string& enroll_plugin, bool force) {
 }
 
 std::string getEnrollSecret() {
-  std::string enrollment_secret;
-  osquery::readFile(FLAGS_enroll_secret_path, enrollment_secret);
-  boost::trim(enrollment_secret);
+  static std::string enrollment_secret;
+
+  if (enrollment_secret.size() == 0) {
+    // Secret has not been read
+    if (FLAGS_enroll_secret_path!= "") {
```

Comment:
nit, add a space between operator and expression.

Run `make format` while your changes are staged to have these style changes automatically applied.